functions/throttle.mjs

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable func-names */
+
+/**
+ * throttle :: a -> Time -> a
+ */
+const throttle = (fn, threshhold) => {
```

Comment:
Typo: threshhold

Reply:
I actually thought I fixed that typo since I noticed it too... weird. It came from Remy Sharp though: https://remysharp.com/2010/07/21/throttling-function-calls 😄